src/display/editor/editor.js

```diff
@@ -1016,8 +1031,24 @@ class AnnotationEditor {
     this.#altText.data = data;
   }
 
+  get guessedAltText() {
+    return this.#altText.guessedText;
+  }
+
+  async setGuessedAltText(text) {
+    await this.#altText.setGuessedText(text);
```

Comment:
It looks like `this.#altText` might be missing given the `#this.altText?` checks below, so shouldn't we also do that here and in the getter just above to make sure the access is safe?